### ISSUE TITLE
Support S3 bucket policy import

### ISSUE THREAD

**Issue by rwolfson** [terraform-providers/terraform-provider-aws#6519]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

It appears that S3 bucket policies bucket policies cannot be imported at this time. It would be nice if they could be. This was hit in AWS provider `v1.45.0`

```
terraform import aws_s3_bucket_policy.sanitized-bucket-name sanitized-bucket-name
aws_s3_bucket_policy.sanitized-bucket-name: Importing from ID "sanitized-bucket-name"...

Error: aws_s3_bucket_policy.sanitized-bucket-name (import id: sanitized-bucket-name): import aws_s3_bucket_policy.sanitized-bucket-name (id: sanitized-bucket-name): resource aws_s3_bucket_policy doesn't support import
```

### New or Affected Resource(s)

* aws_s3_bucket_policy

### Potential Terraform Configuration

The Terraform configuration used to hit this issue was:

```hcl
data "aws_s3_bucket" "sanitized-bucket-name" {
  bucket = "sanitized-bucket-name"
}
data "aws_iam_policy_document" "sanitized-bucket-name" {
  ### Policy contents removed
}

resource "aws_s3_bucket_policy" "sanitized-bucket-name" {
  bucket = "${data.aws_s3_bucket.sanitized-bucket-name.id}"
  policy = "${data.aws_iam_policy_document.sanitized-bucket-name.json}"
}
```

**Comment by bflad** [comment]
Support for `aws_s3_bucket_policy` resource import has been merged in and will release with version 1.47.0 of the AWS provider, sometime later this week. 👍

**Comment by erriapo** [comment]
How can I use the newly merged changes ?

**Comment by bflad** [comment]
Sorry due to re:Invent related activities, I missed commenting with our normal release comment -- hopefully this helps:

This has been released in [version 1.47.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1470-november-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

e.g. if you have existing provider configuration block(s), updating the version attribute and running `terraform init` will upgrade your provider:

```hcl
provider "aws" {
  version = "1.47.0"
}
```

**Comment by erriapo** [comment]
@bflad Thanks for replying. Enjoy re:Invent. It's all good on my side now.
